src/plugins/intel_gpu/src/kernel_selector/cl_kernels/fully_connected_gpu_bf_tiled.cl

```diff
@@ -494,6 +529,8 @@ inline void FUNC(fc_bf_tiled_kernel_default)(
             weights_offset += TILE_K_OFM_PACKED * 2 * SIMD;
             #elif FILTER_LAYOUT_OS_IYX_OSV16 && TILE_OFM == 2 && USE_SLM == 1
             weights_offset += TILE_K_OFM_PACKED / 2 * SIMD;
+            #elif TILE_OFM == 2 && FILTER_LAYOUT_OS_IS_YX_OSV64_ISV2
+            weights_offset += TILE_K_OFM_PACKED * 2 * SIMD;
```

Comment:
Can we refactor this part this like 
TILE_K_OFM_PACKED * OFM_PER_OSV_SIZE * SIMD
and TILE_OFM_PER_OSV_SIZE is calculated and defined as macro?

Reply:
applied